python/metatensor-operations/metatensor/operations/_dispatch.py

```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 
-from ._backend import torch_jit_is_scripting
+# from ._backend import torch_jit_is_scripting
```

Comment:
We'll need this to make it Torchscript-able. We can discuss this next step in person